frontend/Dockerfile

```diff
@@ -17,7 +19,18 @@ ADD nginx.conf /etc/nginx/
 ADD deploy/promis.conf /etc/nginx/
 ADD deploy/promis_api.yaml /var/www/promis/api/
 
-RUN chgrp nginx /etc/ssl/private
-RUN chmod g+x,g+r /etc/ssl/private
+# TODO
+#RUN chgrp nginx /etc/ssl/private
+#RUN chmod g+x,g+r /etc/ssl/private
+
+# Add the actual app
+ADD run_web_promis.sh /
+ADD app /usr/src/promis/app/
+# TODO: please remove what's no longer applicable!
```

Comment:
Add(remove) stuff here if you need more (less) source dirs copied over to the container as the part of a normal build process.